packages/icons/package.json

```diff
@@ -37,34 +37,30 @@
     "@babel/plugin-syntax-jsx": "^7.16.7",
     "@priceline/babel-preset": "workspace:*",
     "@priceline/eslint-config": "workspace:*",
-    "@rushstack/eslint-patch": "^1.3.3",
-    "@storybook/jest": "^0.2.2",
-    "@storybook/testing-library": "^0.2.0",
+    "@rushstack/eslint-patch": "^1.6.0",
+    "@storybook/jest": "^0.2.3",
+    "@storybook/testing-library": "^0.2.2",
     "@svgr/cli": "^5.5.0",
-    "@testing-library/jest-dom": "^6.1.3",
-    "@testing-library/react": "^12.1.5",
+    "@testing-library/jest-dom": "^6.1.5",
+    "@testing-library/react": "^14.1.2",
     "cross-env": "^7.0.3",
-    "eslint": "^8.48.0",
-    "jest": "^29.6.4",
+    "eslint": "^8.55.0",
+    "jest": "^29.7.0",
     "jest-standard-reporter": "^2.0.0",
-    "jest-styled-components": "^7.1.1",
+    "jest-styled-components": "^7.2.0",
     "npm-run-all": "^4.1.5",
     "pcln-design-system": "workspace:*",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-is": "^17.0.2",
-    "react-test-renderer": "^17.0.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-is": "^18.2.0",
+    "react-test-renderer": "^18.2.0",
     "rimraf": "^3.0.2",
     "styled-components": "^5.3.11"
   },
   "peerDependencies": {
     "pcln-design-system": "^5.7.0",
-    "react": ">=16.10.0",
-    "react-dom": ">=16.10.0",
-    "styled-components": ">=5 <5.3.4 || >5.3.7 <6"
-  },
-  "engines": {
-    "node": ">=14",
-    "npm": ">=6.0.0"
```

Comment:
(n-b) There a reason to remove the node/npm engine recs? If the floor is going up, might as well include it...

Reply:
we could, but tbh icons as a package that is distributed doesn't care.

working in the DS repo is governed by our https://github.com/pcln/pcln-web/blob/master/rush.json#L7 nodeSupportedVersionRange